[2024-12-23T09:34:18.687+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-23T09:34:18.697+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_Calendar_NTBK.start_spark scheduled__2024-12-22T00:00:00+00:00 [queued]>
[2024-12-23T09:34:18.704+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_Calendar_NTBK.start_spark scheduled__2024-12-22T00:00:00+00:00 [queued]>
[2024-12-23T09:34:18.705+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-23T09:34:18.714+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): start_spark> on 2024-12-22 00:00:00+00:00
[2024-12-23T09:34:18.721+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=110) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-23T09:34:18.724+0000] {standard_task_runner.py:72} INFO - Started process 112 to run task
[2024-12-23T09:34:18.723+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_Calendar_NTBK', 'start_spark', 'scheduled__2024-12-22T00:00:00+00:00', '--job-id', '205', '--raw', '--subdir', 'DAGS_FOLDER/A_dag_test.py', '--cfg-path', '/tmp/tmp4b1hkh9q']
[2024-12-23T09:34:18.726+0000] {standard_task_runner.py:105} INFO - Job 205: Subtask start_spark
[2024-12-23T09:34:18.762+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_Calendar_NTBK.start_spark scheduled__2024-12-22T00:00:00+00:00 [running]> on host 85f5ad808200
[2024-12-23T09:34:18.812+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_Calendar_NTBK' AIRFLOW_CTX_TASK_ID='start_spark' AIRFLOW_CTX_EXECUTION_DATE='2024-12-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-22T00:00:00+00:00'
[2024-12-23T09:34:18.813+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-23T09:34:18.905+0000] {logging_mixin.py:190} INFO - Error starting Spark Master: 409 Client Error for http+docker://localhost/v1.47/containers/create?name=spark-master: Conflict ("Conflict. The container name "/spark-master" is already in use by container "1c48df6afb7d46b1c969ef12e8024f4e514f5396284da1adb2c0a233ce5f1f42". You have to remove (or rename) that container to be able to reuse that name.")
[2024-12-23T09:34:19.576+0000] {logging_mixin.py:190} INFO - Spark Worker started.
[2024-12-23T09:34:19.583+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-23T09:34:19.637+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-23T09:34:19.642+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_Calendar_NTBK, task_id=start_spark, run_id=scheduled__2024-12-22T00:00:00+00:00, execution_date=20241222T000000, start_date=20241223T093418, end_date=20241223T093419
[2024-12-23T09:34:19.786+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-23T09:34:19.872+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-23T09:34:19.873+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
